src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -405,12 +407,13 @@ identifiers:
 [(8, 10), (8, 13)]  log log
 [(8, 14), (8, 19)]  index index
 [(8, 21), (8, 28)]  element element
-[(11, 6), (11, 42)]
+[(11, 8), (11, 18)]
+[(11, 20), (11, 29)]
```

Comment:
is it okay that these are empty

Reply:
Yes, I think because I'm solving a little issue, after this will be right.

Reply:
I understood these empty lines: when I hover on an expression like this `... = { a: 3 }` I get this error `TypeError: undefined has no properties`. This issue there was also before these changes, but I don't understand why before there was only one line.

Reply:
@marcogallo566 I would recommend you do a rebase first, there are changes on master that fixes this issue.